contracts/mixins/LiquidationHelper.sol

```diff
@@ -5,23 +5,22 @@
 
 pragma solidity 0.5.17;
 
-import "../core/Constants.sol";
 import "@openzeppelin-2.5.0/math/SafeMath.sol";
 
-contract LiquidationHelper is Constants {
+library LiquidationHelper {
     using SafeMath for uint256;
+    uint256 internal constant WEI_PRECISION = 10**18;
+    uint256 internal constant WEI_PERCENT_PRECISION = 10**20;
 
     function _getLiquidationAmounts(
```

Comment:
rename function to remove underscore at the start as it is now a public function

Reply:
fixed